src/components/redux/missions/getMissions.js

```diff
@@ -0,0 +1,14 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+const url = ' https://api.spacexdata.com/v3/missions';
+const getMissions = createAsyncThunk('missions/getMissions', async () => {
+  try {
+    const response = await axios.get(url);
+    return response.data;
+  } catch (error) {
+    return error;
+  }
+});
+
```

Comment:
- [ ] Kindly filter the data to be saved from the API. It's not all items from the API that should be stored. Kindly visit [this requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/group-project/project_space_travelers_hub.md#redux-fetch-data-and-update-redux-store) and see the items to be saved.